src/main/kotlin/no/nav/familie/ba/sak/kjerne/forrigebehandling/EndringIVilkårsvurderingUtil.kt

```diff
@@ -78,9 +83,16 @@ object EndringIVilkårsvurderingUtil {
                 val erEndringerIRegelverk = nåværende.vurderesEtter != forrige.vurderesEtter
                 val erVilkårSomErSplittetOpp = nåværende.periodeFom != forrige.periodeFom
 
+                val erEndringIFinnmarkstillegg = nåværende.finnmarkEllerSvalbardVilkår() != forrige.finnmarkEllerSvalbardVilkår()
+                val erKunEndringIFinnmarkstillegg = erEndringIFinnmarkstillegg && !erEndringerIUtdypendeVilkårsvurdering && !erEndringerIRegelverk
+
                 (forrige.obligatoriskUtdypendeVilkårsvurderingErSatt() && erEndringerIUtdypendeVilkårsvurdering) ||
                     erEndringerIRegelverk ||
-                    erVilkårSomErSplittetOpp
+                    if (featureToggleService.isEnabled(FeatureToggle.SLÅ_SAMMEN_FINNMARK_ELLER_SVALBARD)) {
+                        (erVilkårSomErSplittetOpp && !erKunEndringIFinnmarkstillegg)
+                    } else {
+                        erVilkårSomErSplittetOpp
+                    }
```

Comment:
Koden kan forenkles litt. Da kan vi slette funksjonen `VilkårResultat.finnmarkEllerSvalbardVilkår()`
```suggestion
                val erKunEndringIFinnmarkstillegg = !erEndringerIUtdypendeVilkårsvurdering && !erEndringerIRegelverk

(forrige.obligatoriskUtdypendeVilkårsvurderingErSatt() && erEndringerIUtdypendeVilkårsvurdering) ||
erEndringerIRegelverk ||
                    if (featureToggleService.isEnabled(FeatureToggle.SLÅ_SAMMEN_FINNMARK_ELLER_SVALBARD)) {
                        (erVilkårSomErSplittetOpp && !erKunEndringIFinnmarkstillegg)
                    } else {
                        erVilkårSomErSplittetOpp
                    }
```